### DISCUSSION TITLE
完成支付购买专业版，并未延长专业版时长，也未转到专业版界面

### DISCUSSION THREAD

**Discussion by dreadeath1** [getlantern/lantern-forum-cn#912] [Q&A]
@Mike777Khoo 
![Uploading QQ截图20240928095140.png…]()
![Uploading 94bce35f1fde33d5b79afc1d7e72b5f3.jpg…]()
账号Dreadeath@live.com

请尽快核实解决

**Comment by dreadeath1** [comment]
@Mike777Khoo
管理员您好请尽快核实解决

**Comment by MaxMKMax** [comment]
@dreadeath1 好的. 我已经帮您提交问题上去了. 我无法帮您核实, 没权限. 需负责人帮您弄.

**Comment by dreadeath1** [comment]
请问解决进度如何了，届时这边是您发送激活码过来还是直接在终端上延长会员时长？
还有麻烦留意一下我这边还有提前预定还有附赠的3个月的试用期
________________________________
发件人: Mike K ***@***.***>
发送时间: 2024年10月1日 18:13
收件人: getlantern/lantern-forum-cn ***@***.***>
抄送: dreadeath1 ***@***.***>; Mention ***@***.***>
主题: Re: [getlantern/lantern-forum-cn] 完成支付购买专业版，并未延长专业版时长，也未转到专业版界面 (Discussion #912)


@dreadeath1<https://github.com/dreadeath1> 好的. 我已经帮您提交问题上去了. 我无法帮您核实, 没权限. 需负责人帮您弄.

―

**Comment by dreadeath1** [comment]
请问解决进度如何了
________________________________
发件人: Mike K ***@***.***>
发送时间: 2024年10月1日 18:13
收件人: getlantern/lantern-forum-cn ***@***.***>
抄送: dreadeath1 ***@***.***>; Mention ***@***.***>
主题: Re: [getlantern/lantern-forum-cn] 完成支付购买专业版，并未延长专业版时长，也未转到专业版界面 (Discussion #912)


@dreadeath1<https://github.com/dreadeath1> 好的. 我已经帮您提交问题上去了. 我无法帮您核实, 没权限. 需负责人帮您弄.

―
